ui/nano.html

```diff
@@ -552,20 +634,38 @@
                 displayName: plugin.displayName,
                 profile: '',
                 version: '',
-                plugin,
+                pluginRef: plugin,
```

Comment:
This breaks a pattern within grid-ui. It is admittedly ugly, but grid-ui is littered with plugin parsing, like `plugin.plugin.config`, for example. `plugin.pluginRef` is better, but I dont love it. Main point: whatever we use should be updated in grid-ui, too. Or abstracted better.

Reply:
yeah, I've noticed this. the main problem seems to be that we have different interfaces for the plugins, from grid's `PluginHost` to grid's provider to the UI (`preload.js` and `preload-webview.js`) to the store in the UI.

in `grid-ui` we may be able to solve this by renaming some parts to pluginStore and pluginRef can always mean the original interface provided from grid's `PluginHost`.

Reply:
:shipit: